libs/rpcsvc-proto/patches/0001-po-update-for-gettext-0.22.patch

```diff
@@ -0,0 +1,26 @@
+From 600879af780453bfe86e1e75f7a9f8a959982f29 Mon Sep 17 00:00:00 2001
+From: Vladimir Ermakov <vooon341@gmail.com>
+Date: Thu, 31 Jul 2025 09:36:47 +0200
+Subject: [PATCH] po: update for autoconf gettext macros 0.22
+
+Fix build error:
+```
+*** error: gettext infrastructure mismatch: using a Makefile.in.in from gettext version 0.20 but the autoconf macros are from gettext version 0.22
+```
+
+Signed-off-by: Vladimir Ermakov <vooon341@gmail.com>
+---
+ po/Makefile.in.in | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+--- a/po/Makefile.in.in
++++ b/po/Makefile.in.in
+@@ -8,7 +8,7 @@
+ # without any warranty.
+ #
+ # Origin: gettext-0.20.2
```

Comment:
cc: @thkukuk can you please look at it?

Reply:
The gettext.m4 macro and po/Makefile.in.in from rpcsvc-proto-1.4.4.tar.xz both are from gettext-0.20.2 and work fine together.
So if you have a gettext version mismatch here, your build process is broken, it updates the macro but does not regnerate Makefile.in.in.
I will not accept such a patch as it breaks the current package, with this patch there is really a gettext version mismatch.

Reply:
@thkukuk So is there any better solution?

Reply:
Fix the problem which introduces the gettext version mismatch in the openwrt build system?
I don't know the build system, so I don't know how this version mismatch happened and how to solve that.
But until now this is the only report about such a problem, no other distribution has that and the distributions I checked don't need any patches.

Reply:
I guess the reason is that we always prefer the m4 files from our own copy and result gettext version mismatch. It's more or less similar to https://github.com/openwrt/openwrt/issues/19601#issuecomment-3137320157.